src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { Fragment, useState } from 'react';
+import './styles/index.css';
+import { useIntl } from 'react-intl';
+import SafeRedirect from '../../../SafeRedirect';
+
+interface IFreeAccountProps {}
+
+const FreeAccount: React.SFC<IFreeAccountProps> = (props) => {
+  const intl = useIntl();
+  const _ = (id: string, values: any) => intl.formatMessage({ id: id }, values);
+
+  const [redirect, setRedirect] = useState(false);
+
+  const tittle = (
+    <div>
+      <span className="data-studio-logo"></span>
+      <h1>{_('big_query.free_title', '')}</h1>
+    </div>
+  );
+
+  const list = (
+    <div>
+      <ul className="dp-list-detail">
+        <li>
+          <span className="check-icon"></span>
+          <span className="list-item-span">{_('big_query.free_ul_item_one', '')}</span>
+        </li>
+        <li className="list-item">
+          <span className="check-icon"></span>
+          <span className="list-item-span">{_('big_query.free_ul_item_two', '')}</span>
+        </li>
+      </ul>
+    </div>
+  );
+
+  const btn = (
+    <button
+      type="button"
+      className="dp-button button-medium primary-green"
+      onClick={() => setRedirect(true)}
+    >
+      {_('big_query.free_btn_redirect', '')}
+    </button>
+  );
+
+  if (redirect) {
+    return <SafeRedirect to="/ControlPanel/AccountPreferences/UpgradeAccount" />;
+  }
+
+  return (
+    <Fragment>
+      <section className="dp-gray-page p-t-54 p-b-54">
+        <div>
+          <div className="col-md-6">
+            {tittle}
+            <p>{_('big_query.free_text_summary', '')}</p>
+            {list}
+            <p>
+              <strong>{_('big_query.free_text_strong', '')}</strong>
+            </p>
+            <div className="line"></div>
+            <div>{btn}</div>
```

Comment:
Something like this:

```suggestion
             <div>
               <a 
                 href="/ControlPanel/AccountPreferences/UpgradeAccount" 
                 className="dp-button button-medium primary-green"
               >
                 {_('big_query.free_btn_redirect', '')}
               </a>
             </div>
```

Reply:
Change applied!!

Reply:
👍 resolving the conversation...